backend/ml_tooling/llm/config/models.yaml

```
@@ -0,0 +1,75 @@
+
+  huggingface:
+    # HuggingFace models typically need api_base in kwargs (set per-model)
+    llm_inference_kwargs: {}
+    supported_models:
+      "huggingface/unsloth/llama-3-8b":
+        llm_inference_kwargs:
+          api_base: "https://api-inference.huggingface.co/models/unsloth/llama-3-8b"
+      "huggingface/mistralai/Mixtral-8x22B-v0.1":
+        llm_inference_kwargs:
+          api_base: "https://api-inference.huggingface.co/models/mistralai/Mixtral-8x22B-v0.1"
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)